src/Data/StateVar.hs

```diff
@@ -8,7 +8,7 @@
 #endif
 {-# LANGUAGE TypeFamilies #-}
 -- Foreign.ForeignPtr is unsafe before GHC-7.10
-#if __GLASGOW_HASKELL__ >=710
+#if __GLASGOW_HASKELL__ >=710 && MIN_VERSION_base(4,8,0)
```

Comment:
`>= 704`, as `{-# LANGUAGE Safe #-}` works since then. (If we want to assume that base and ghc versions are independent, which is fair).

Reply:
Done.